src/ui/osx/TogglDesktop/test2/ProjectTextField.h

```diff
@@ -12,6 +12,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 @class TimeEntryViewItem;
 @class AutocompleteItem;
+@class IdleEvent;
```

Comment:
It's really good to use @class on the header rather importing the entire file on the header 👍